### PR TITLE
feat(integrations): add API rate limiting to Front

### PR DESCRIPTION
**Pull request by hassan254-prog**
Add rate limiting to both the documentation and the `providers.yaml`.

### CODE REVIEW

File: docs-v2/integrations/all/front/connect.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/front/connect.mdx]: This is not related to Connect UI.
Connect.mdx should only be used to guide end user, so only doc related to Connect UI

Comment (hassan254-prog) [reply docs-v2/integrations/all/front/connect.mdx]: Aah yes 🤦‍♂️, updated with the latest [commit](https://github.com/NangoHQ/nango/pull/3101/commits/bf4813f43df99624426ab34847dd5b9140aa8a4f). Thank you.

**Review by bodinsamuel — APPROVED** [review]
